src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 700;
+  font-style: normal;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.bigForm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.bigForm img {
+  position: absolute;
```

Comment:
we don't use camelCase in CSS classnames, we use `kebab-case` or BEM
do not style tags, style classes